src/coreclr/pal/src/thread/process.cpp

```diff
@@ -1675,6 +1675,7 @@ GetProcessIdDisambiguationKey(DWORD processId, UINT64 *disambiguationKey)
     {
         TRACE("GetProcessIdDisambiguationKey: getline() FAILED");
         SetLastError(ERROR_INVALID_HANDLE);
+        free(line);
         return FALSE;
     }
 
```

Comment:
It looks like https://github.com/dotnet/runtime/pull/74899/files#diff-236bca3ea8ce55b54c55c47ab9d21a4db9596eb3fbc6052565a81a2ebb3ee73fR1701-R1702 should also be moved up to before the `if (sscanfRet != 1)` check?

Reply:
Right. Fixed. :)